src/paho/mqtt/client.py

```diff
@@ -3241,24 +3555,24 @@ def _handle_connack(self):
                                     m.dup,
                                     properties=m.properties
                                 )
-                            if rc != 0:
+                            if rc != MQTTErrorCode.MQTT_ERR_SUCCESS:
                                 return rc
                         elif m.state == mqtt_ms_resend_pubrel:
                             self._inflight_messages += 1
                             m.state = mqtt_ms_wait_for_pubcomp
                             with self._in_callback_mutex:  # Don't call loop_write after _send_publish()
                                 rc = self._send_pubrel(m.mid)
-                            if rc != 0:
+                            if rc != MQTTErrorCode.MQTT_ERR_SUCCESS:
                                 return rc
                     self.loop_write()  # Process outgoing messages that have just been queued up
 
             return rc
         elif result > 0 and result < 6:
-            return MQTT_ERR_CONN_REFUSED
+            return MQTTErrorCode.MQTT_ERR_CONN_REFUSED
         else:
-            return MQTT_ERR_PROTOCOL
+            return MQTTErrorCode.MQTT_ERR_PROTOCOL
 
-    def _handle_disconnect(self):
+    def _handle_disconnect(self) -> "Literal[MQTTErrorCode.MQTT_ERR_SUCCESS]":
```

Comment:
If the only value this function can return is this, should it just return None instead?

Reply:
Yes, I didn't changed them because it would break the signature (breaking change)... but it's a private function, we are not going to have backward compatibility with private function :)